### PR TITLE
feat: Fixes Search Input and Combobox + makes them better

### PR DESCRIPTION
**Pull request by ChristianKienle**

### CODE REVIEW

File: ui/src/components/SearchInput/SearchInput.vue [hunk 1]
```diff
@@ -1,107 +1,110 @@
...
-      this.emitSearch();
+    hasMatches() {
+      return this.completionsMatchingPredicate.length > 0;
     },
-    setCurrentValue(newValue: string) {
-      this.searchValue = newValue;
-      this.emitSearch();
-      this.$emit("update:value", this.searchValue);
+    normalizedPredicate() {
+      console.log(this.predicate);
```
Comment (CodesOfRa) [inline ui/src/components/SearchInput/SearchInput.vue]: can you remove the `console.log` ? :D

Comment (ChristianKienle) [reply ui/src/components/SearchInput/SearchInput.vue]: I will remove that and add the linter rule so that this does not happen again. :D

**Comment by ChristianKienle** [comment]
thanks for the review. fixed everything.

File: ui/src/components/SearchInput/SearchInput.vue (hunk 1 above)
Comment (bcullman) [reply ui/src/components/SearchInput/SearchInput.vue]: 💯

**Pull request merged**